src/widget/wsearchlineedit.cpp

```diff
@@ -146,14 +161,29 @@ void WSearchLineEdit::setup(const QDomNode& node, const SkinContext& context) {
 
 void WSearchLineEdit::resizeEvent(QResizeEvent* e) {
     QLineEdit::resizeEvent(e);
-    QSize sz = m_clearButton->sizeHint();
-    int frameWidth = style()->pixelMetric(QStyle::PM_DefaultFrameWidth);
-    int height = (rect().bottom() + 1 - sz.height()) / 2;
+    // ToDo This just reads some default Qt frame width. For me it always retiurns '2'
+    // int frameWidth = style()->pixelMetric(QStyle::PM_DefaultFrameWidth);
```

Comment:
it looks like you still need to figure this part out

Reply:
to me it's unclear what this is supposed to do. The documentation doesn't help me much, and I don't see how this affects the display in skins https://doc.qt.io/qt-5/qstyle.html#PixelMetric-enum > `QStyle::PM_DefaultFrameWidth`
IMO only the styles in the official skins are relevant for us.

Reply:
The only [explanation I found](http://www.informit.com/articles/article.aspx?p=1405556&seqNum=2):
> PM_DefaultFrameWidth defines the width of the line used around QFrame, QPushButton, QSpinBox, and many other widgets. For other PM_ xxx values, we inherit the pixel metric value from the base class.

So I assume we don't need that because we style the skins anyway.

Reply:
removed.